server/service/src/sync/api/core.rs

```diff
@@ -238,6 +242,30 @@ async fn response_or_err(
     Err(SyncApiErrorVariantV5::from_response_and_status(response.status(), response).await)
 }
 
+// OMS Central does not yet do auth validation for site credentials
+// So we call Legacy central server for this
+// (Use sync API for simplest auth)
+pub async fn validate_site_auth(
```

Comment:
This little refactor as one commit probably makes these changes hard to cherry pick into old releases' patches. Oh well...

Reply:
That's ok, i like the refactor, plus this will be the only version it goes to, I've rebased to 2.7.3 (as that's the next release, coming hot)